core/Symbols.cc

```diff
@@ -886,12 +886,8 @@ bool SymbolRef::isUnderNamespace(const GlobalState &gs, ClassOrModuleRef otherCl
 
 vector<ClassOrModule::FuzzySearchResult>
 ClassOrModule::findMemberFuzzyMatchConstant(const GlobalState &gs, NameRef name, int betterThan) const {
-    // Performance of this method is bad, to say the least.
-    // It's written under assumption that it's called rarely
-    // and that it's worth spending a lot of time finding a good candidate in ALL scopes.
-    // It may return multiple candidates:
-    //   - best candidate per every outer scope if it's better than all the candidates in inner scope
-    //   - globally best candidate in ALL scopes.
+    // This function is somewhat expensive, as it will crawl all owning scopes to determine if there's a reasonable
```

Comment:
```suggestion
    // This function is somewhat expensive, as it will crawl all owners to determine if there's a reasonable
```

Slight wording change, it's hard for me to hear "scopes" and not think lexical scopes or module nesting scopes (even though owning scopes is valid). Not blocking.

Reply:
I think saying owners here is reasonable, I'll make that change 👍